rb/lib/selenium/webdriver/remote/http/common.rb

```diff
@@ -22,7 +22,7 @@ module Http
         class Common
           MAX_REDIRECTS   = 20 # same as chromium/gecko
           CONTENT_TYPE    = 'application/json'.freeze
-          DEFAULT_HEADERS = {'Accept' => CONTENT_TYPE}.freeze
+          DEFAULT_HEADERS = {'Accept' => CONTENT_TYPE, 'Content-Type' => 'application/x-www-form-urlencoded'}.freeze
```

Comment:
Does it send requests with urlencoded bodies anywhere? I thought it sends only json. Maybe content-type should be `application/json` by default?

Reply:
@kolen If you have `$VERBOSE` set to true, you'll see that Net-HTTP has been setting the Content-Type header to `application/x-www-form-urlencoded` automatically (and emitting warnings about it if `$VERBOSE` is true). As I mentioned in #5615 - I think `application/json` is more correct, but I am not in a position to tell if you if "more correct" >= "the way it has been historically".